packages/gestalt/src/__snapshots__/Row.test.js.snap

```diff
@@ -2,172 +2,44 @@
 
 exports[`Row aligns items 1`] = `
 <div
-  className="box marginEnd0 marginStart0"
-  style={
-    Object {
-      "height": undefined,
-      "width": undefined,
-    }
-  }
+  className="itemsEnd row horizontalGap0 horizontalGap0 > *"
```

Comment:
@dangerismycat same for horizontalGap, I think we can remove the `horizontalGap0 > *`

Reply:
agreed, removed